pkg/network/infraconfigurators/masquerade_test.go

```diff
@@ -264,18 +270,32 @@ var _ = Describe("Masquerade infrastructure configurator", func() {
 					mockNetfilterIPTables,
 					iptables.ProtocolIPv6),
 				table.Entry("NFTables backend on a dual stack cluster when *reserved* ports are specified",
-					newVMIMasqueradeInterface(namespace, vmName, getReservedPortList()...),
+					newVMIMasqueradeInterface(namespace, vmName, getReservedPortList(migrationOverTCP)...),
 					mockNetfilterNFTables,
 					iptables.ProtocolIPv6),
 				table.Entry("NFTables backend on a dual stack cluster when using an ISTIO aware VMI",
 					newIstioAwareVMIWithSingleInterface(namespace, vmName),
 					mockNetfilterNFTables,
 					iptables.ProtocolIPv6),
+				table.Entry("NFTables backend on an IPv4 cluster with migration over sockets",
+					newVMIMasqueradeMigrateOverSockets(namespace, vmName, getReservedPortList(!migrationOverTCP)...),
+					mockNetfilterNFTables),
 			)
 		})
 	})
 })
 
+func portsUsedByLiveMigration(isMigrationOverSockets bool) []string {
+	if isMigrationOverSockets {
+		return []string{}
```

Comment:
nit: I would expect this to be `nil`.

And there is no need for the `else` part.

Reply:
I wrote it this way to demonstrate both cases. I don't mind changing this.